Description: Manually remove the lt- prefix that appears in some manpages.
Author: Nicolas Boullis <nboullis@debian.org>
--- a/frontends/cli/vcd-info.1
+++ b/frontends/cli/vcd-info.1
@@ -3,7 +3,7 @@
 .SH NAME
 vcd-info \- manual page for vcd-info 0.7.24
 .SH SYNOPSIS
-.B lt-vcd-info
+.B vcd-info
 [\fIOPTION\fR...]
 .SH DESCRIPTION
 .TP
--- a/frontends/xml/vcdxrip.1
+++ b/frontends/xml/vcdxrip.1
@@ -3,7 +3,7 @@
 .SH NAME
 vcdxrip \- manual page for vcdxrip 0.7.24
 .SH SYNOPSIS
-.B lt-vcdxrip
+.B vcdxrip
 [\fIOPTION\fR...]
 .SH DESCRIPTION
 .TP
